rollup/rollup-cdn.config.js

```diff
@@ -23,32 +23,49 @@ const sourceDir = 'src';
 const outputDir = 'dist-cdn';
 
 export default {
+  // defines the files to be included into the build: all components + smart manager + translations
   input: inputs(sourceDir, (file) => {
     const { name: entryPath } = path.parse(file);
     return [entryPath, file];
   }),
   output: {
     dir: outputDir,
     sourcemap: true,
+    // add a hash on every file
     entryFileNames: '[name]-[hash].js',
+    // sometimes, chunks are so small that they become counterproductive. Here we force some modules to not generate chunks.
```

Comment:
// sometimes, chunks are so small that they become counterproductive. Here we force some modules to be grouped in the same chunk.